src/content/getting-started/cloud-provider-accounts/vintage/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+linkTitle: Cloud provider accounts for Vintage
+title: Cloud provider accounts for Vintage
+description: How to set up your AWS account or your Azure subscription in Vintage in order to run Giant Swarm management clusters and workload clusters under your jurisdiction.
+weight: 80
+menu:
+  main:
+    identifier: gettingstarted-infraprovider-vintage
+    parent: gettingstarted-infraprovider
+last_review_date: 2023-11-27
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+---
```

Comment:
I guess this was moved, so it should have `aliases:` pointing to the previous URL